animareflection/ui
# Vercel CI

## Description

##### Task link: https://trello.com/c/5BZJkwBd/293-vercel-ci-migration-github-actions

- Added Vercel config manifest (`vercel.json`)
- Added GitHub Actions deployment workflow

Behind the scenes, here are the steps I took:
- Added Vercel deployment token, Vercel organization ID, and Vercel project ID secrets to this repo
- Linked git repo to Vercel project with `vercel` CLI (this is supposed to work for new projects as well; existing Vercel project not required)

## Test Steps

- Verify deployment is correctly deployed (see Actions comment with "Deploy preview for `$PROJECT_NAME` ready!")
